src/components/pages/Cases/CaseTable.js

```diff
@@ -13,6 +13,9 @@ import Icon from '@ant-design/icons';
 
 import { Table, Space, Button, Input, Tabs } from 'antd';
 import './CaseTable.less';
+import CaseDetails from '../CaseOverview/CaseDetails';
+import { TableCell } from '@material-ui/core';
+import { TableRow } from '@material-ui/core';
```

Comment:
The Case Table is already made with Ant D, can you explain why you went with MUI again?